content/maxwell3_fdem/sphere_freespace.rst

```diff
@@ -0,0 +1,262 @@
+.. _sphere_freespace:
```

Comment:
Great figures, great explanations, great job!
Please see the comments below for adding some values and details to it before merging